src/bluebottle.cuh

```diff
@@ -250,4 +250,7 @@ extern __constant__ real _node_p[NNODES];
  *  phi location of nodes
  */
 
+extern __constant__ int _s_mm[64];
+extern __constant__ int _s_nn[64];
```

Comment:
Can you leave a comment here about what these coeffs are? Also, what's the reason for initializing this to a static size of 64?

Reply:
These are the m's and n's used in P_nm and N_nm when calculating lebedev quadrature of temperature field around particles. 64 is the total number of coefficients for a 7-th order expansion. They are similar to _mm and _nn, but since temperature field uses a different expansion, so they are defined separately.